AlphaCentauri/Assets/Scripts/RodCasting.cs

```diff
@@ -28,13 +32,66 @@ void BobberThrow()
         float time = distance.magnitude / bobberVelocity;
         rigidbody.velocity = bobberVelocity * distance.normalized + new Vector3(0, time * Physics.gravity.magnitude * 0.5f);
         // bobberClone.position = target.position;
+        StartFishing();
+        //Trigger the fish eat the bait
     }
     void SetPowerLevel()
     {
         amplitude = horizontalBar.parent.GetComponent<RectTransform>().rect.width - horizontalBar.GetComponent<RectTransform>().rect.width;
         playHorizontal = true;
     }
     // Tween the bar of rightLeft or upDown
+    void StartFishing()
+    {
+        StartCoroutine(FishingCorountine());
+    }
+    IEnumerator FishingCorountine()
+    {
+        if (IsFishing)
+        {
+            int wait_time = Random.Range (5, 10);//Wait random time to fish to bite the bait
```

Comment:
kinda nit pick on this one but js as a naming convention ini harusnya camelCase buts its fine lool